python.md

```diff
@@ -280,3 +280,9 @@ source:
 ```
 python3 -m pip install --user --no-binary confluent-kafka confluent-kafka
 ```
+
+### open3d
+
+Open3d required glibc version 2.27 or higher. Amazon Linux 2 include glibc 2.26, which is not sufficient. In order to
+use open3d, please use Amazon Linuxx 2023 or later, Ubuntu Bionic (18.04) or later, or another supported distribution.
+See [open3d documentation](http://www.open3d.org/docs/release/getting_started.html) for more information.
```

Comment:
Do you think we should also suggest installing glibc 2.27+ versions as a work around for people who want to still use AL2?

Reply:
That would not be my first choice.